src/main/play-doc/operation/Install.md

```diff
@@ -276,10 +373,17 @@ We will limit the bundle's sudo privileges to running `/usr/bin/reloadHAProxy.sh
 [172.17.0.1]$ echo "conductr-agent ALL=(root) NOPASSWD: /usr/bin/reloadHAProxy.sh" | sudo tee -a /etc/sudoers
 ```
 
-On RHEL and CentOS it may also be neccessary to [disable default requiretty](https://bugzilla.redhat.com/show_bug.cgi?id=1020147) for the `conductr-agent` user in `sudoers`.
+#### Using HAProxy reload script from a customised location
+
+HAProxy reload script location can be customised. ConductR-HAProxy will install its reload script in this location upon startup.
```

Comment:
customized instead of customised (We generally use american english in our docs)